generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -22,7 +23,7 @@ public CheckpointOperation(State state) {
     @Override
     public void execute() {
         try {
-            MultiCheckpointWriter<CorfuTable<String, String>> mcw = new MultiCheckpointWriter<>();
+            MultiCheckpointWriter<PersistentCorfuTable<String, String>> mcw = new MultiCheckpointWriter<>();
```

Comment:
`ICorfuTable` should be used here so that both interface implementations can be tested.